src/index.html

```diff
@@ -3,15 +3,180 @@
+  
+        <label class="age label-first-form-bottom">
+          How old are You?
+          <input 
+            type="number"
+            name="age" 
+            min="1"
+            max="100" 
+            value="12"
+            required
+          >
+        </label>
+  
+        <label class="birthday label-first-form-bottom">
+          Full date of birth:
+          <input 
+            type="date"
+            name="birthday" 
+            required
+          >  
+        </label>
+  
+        <label class="agreement">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="agreement"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="second-form">
+        <legend>Registration</legend>
+
+        <label class="email label-second-form-bottom">
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+            required 
+          >
+        </label>
+
+        <label class="password">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="third-form">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="cats label-third-form-bottom">
+          Do you love cats?
+          <input 
+            type="radio" 
+            name="cats"
+            required
+          > Yes
+          <input 
+            type="radio" 
+            name="cats"
+          > No
+        </label>
+
+        <label class="favorite-color label-third-form-bottom">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color"
+            required
+          >
+        </label>
+
+        <label class="go-to-bed label-third-form-bottom">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time"
+            required 
+            step="2"
+          >
+        </label>
+
+        <label class="brand-of-cars label-third-form-bottom">
+          What are your favorite brand of cars?
+          <select 
+            class="select-cars" 
+            multiple 
+            required
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="rate-our-work label-third-form">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="range"
+            autocomplete="off" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="last-form">
+        <legend>Additional info</legend>
+
+        <label class="comments label-last-form-bottom">
+          Comments:
```

Comment:
checklist p.8 - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.

<img width="326" alt="Screenshot 2023-02-20 at 16 13 47" src="https://user-images.githubusercontent.com/104769168/220134281-beedfeb5-8627-42d7-8fb9-58f2c0834ea9.png">